ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentService.scala

```diff
@@ -108,4 +136,20 @@ class DeploymentService(processRepository: FetchingProcessRepository[Future],
 
   private def toManagerUser(loggedUser: LoggedUser) = User(loggedUser.id, loggedUser.username)
 
+  private def notifyOfDeploymentAction(processIdWithName: ProcessIdWithName,
```

Comment:
since we return original action from this method, I would name this method somehow differently, eg. `withDeploymentActionNotification` or `withNotificationSideEffect`

Reply:
:+1: